test/e2e/instrumentation/monitoring/prometheus.go

```diff
@@ -1,388 +0,0 @@
-/*
-Copyright 2018 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package monitoring
-
-import (
-	"context"
-	"encoding/json"
-	"fmt"
-	"math"
-	"time"
-
-	"github.com/prometheus/common/model"
-
-	"github.com/onsi/ginkgo"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	clientset "k8s.io/client-go/kubernetes"
-	"k8s.io/kubernetes/test/e2e/common"
-	"k8s.io/kubernetes/test/e2e/framework"
-	instrumentation "k8s.io/kubernetes/test/e2e/instrumentation/common"
-)
-
-const (
-	prometheusQueryStep                = time.Minute
-	prometheusMetricErrorTolerance     = 0.25
-	prometheusMetricValidationDuration = time.Minute * 2
-	prometheusRate                     = time.Minute * 2
-	prometheusRequiredNodesUpDuration  = time.Minute * 5
-	prometheusService                  = "prometheus"
-	prometheusSleepBetweenAttempts     = time.Second * 30
-	prometheusTestTimeout              = time.Minute * 5
-	customMetricValue                  = 1000
-	targetCPUUsage                     = 0.1
-)
-
-var _ = instrumentation.SIGDescribe("[Feature:PrometheusMonitoring] Prometheus", func() {
```

Comment:
Do all of these tests depend strictly on the cluster addon from the `cluster/`?
If prometheus was to be enabled using some method other than kube-up, should this be still removed?

Reply:
Those tests where strictly developed for cluster addon in `cluster`. There is no officially supported  prometheus distribution by kubernetes community.